<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_tongji</class>
 <widget class="QDialog" name="win_tongji">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>统计</string>
  </property>
  <widget class="QTableWidget" name="zonglan_table">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>301</width>
     <height>431</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>75</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>时间（年-月）</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>销售量（本）</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>销售额（元）</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="paihangbang_table">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>301</width>
     <height>431</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>75</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>书目ID</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>销售量（本）</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>销售额（元）</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="hint_zonglan_table">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>销售情况总览</string>
   </property>
  </widget>
  <widget class="QLabel" name="hint_paihangbang_table">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>具体时间排行榜</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
